### DISCUSSION TITLE
openings/vents to outside : pressure (simulating wind)

### DISCUSSION THREAD

**Discussion by jurvanoerle** [firemodels/cfast#2135] [Q&A] [answered]
Hi,
I am using Cfast 7.7.4 and I can't find the way to simulate wind effect on a vent to outside by prescribing an (over/under) pressure.
(in very old version of Cfast a windvelocity could be prescribed, and the angle resp winddirection of openings/vents, in current version ). 
Did I miss something? Regards, Jur

**Comment by drjfloyd** [comment, marked as answer]
Old versions of CFAST had a number of submodels that had poor or no verifcation and validation or other issues. Many have been removed to simplify the long term maintainability of CFAST.
